harness/determined/cli/notebook.py

```diff
@@ -100,12 +101,12 @@ def open_notebook(args: Namespace) -> None:
             Arg("task_id", help="notebook ID", metavar="notebook_id"),
```

Comment:
Looks like the `logs` command above this line should also get the same treatment.